modules/simphera_aws_instance/postgresql.tf

```diff
@@ -1,3 +1,12 @@
+resource "aws_db_subnet_group" "database" {
+  name       = "${var.infrastructurename}-vpc"

```

Comment:
Very important: Use `name = "${local.instancename}-vpc"` Otherwise you will get a conflict when you have multiple simphera instances defined in `var.simpheraInstances`

Reply:
Hm, ok, makes sense, but not sure of any case where multiple simphera instances are defined. In any case, will update this.

Reply:
Done

Reply:
Indeed, the feature is currently barely used. The idea behind it is to have different SIMPHERA stages that share the same EKS etc. It was adopted from the Azure reference architecuture. I expect the feature to become more important in the future.